app/src/lib/shells/darwin.ts

```diff
@@ -47,11 +47,24 @@ function getBundleID(shell: Shell): string {
   }
 }
 
+async function getOldAlacrittyShellPath(): Promise<string | null> {
+  const oldBundleId = 'io.alacritty'
+  try {
+    return await appPath(oldBundleId)
+  } catch (e) {
+    return null
+  }
+}
+
```

Comment:
I think the best way to handle this is to make `getBundleID` return a list of bundleIDs instead (we did something similar with editors). In this case, all other shells will return an array with just 1 item, except for Alacritty which will return an array with 2 items.

Then the rest of the code will need to iterate through all the bundle IDs in the list.

Reply:
@sergiou87 How about this call?

https://github.com/desktop/desktop/blob/bcc7ffc2857f13fec4f9be2784ff9f977e76b60b/app/src/lib/shells/darwin.ts#L174-L175

Reply:
Good question. Maybe it makes sense to have a `IFoundDarwinShell` that extends `IFoundShell` by adding a `bundleID` field. Then `getShellPath` will return not only the path, but also the bundle ID that was obtained to retrieve such path.

Does that make sense? 😳

Reply:
Nega meni sharhim o‘chirildi